operator/duties/attester.go

```diff
@@ -77,21 +76,12 @@ func (h *AttesterHandler) HandleDuties(ctx context.Context) {
 			buildStr := fmt.Sprintf("e%v-s%v-#%v", currentEpoch, slot, slot%32+1)
 			h.logger.Debug("🛠 ticker event", zap.String("epoch_slot_pos", buildStr))
 
-			if h.fetchFirst {
-				h.fetchFirst = false
+			h.processExecution(currentEpoch, slot)
+			if h.indicesChanged {
+				h.duties.ResetEpoch(currentEpoch)
 				h.indicesChanged = false
-				h.processFetching(ctx, currentEpoch, slot)
-				// TODO: (Alan) genesis support
-				//h.processExecution(currentEpoch, slot)
-			} else {
-				// TODO: (Alan) genesis support
-				//h.processExecution(currentEpoch, slot)
-				if h.indicesChanged {
-					h.duties.ResetEpoch(currentEpoch)
-					h.indicesChanged = false
-				}
-				h.processFetching(ctx, currentEpoch, slot)
```

Comment:
i dont see the point of removing these comments as we're gonna need them in a week or so anyways 😅